contracts/QuestFactory.sol

```diff
@@ -2,8 +2,9 @@
 pragma solidity ^0.8.15;
 
 import '@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol';
-import {Erc20Quest} from './Erc20Quest.sol';
 import {IQuestFactory} from './interfaces/IQuestFactory.sol';
+import {Quest as QuestContract} from './Quest.sol';
```

Comment:
why import the base contract? You can call all these functions from the `Erc20Quest` contract right?

Reply:
I think I was just worried for 1155 quests lol (even though they are quite similar)

I can change to `Erc20Quest`